README-agile.md

```diff
@@ -0,0 +1,145 @@
+# AGILE scripts
+
+This repository contains start and stop scripts for various [AGILE](http://agile-iot.eu/) components.
+Currently, AGILE is tested on Raspberry PI 2/3 Model B running Raspbian, and on x86_64 running Ubuntu 16.04.
+Since we use docker containers, it should also run in many other environment.
+
+## Requirements
+
+### On Raspberry PI 2/3 Model B running Raspbian
```

Comment:
I think we need to rewrite. At the moment I was just adding the old file, which is my preferred way of merging files between repos: First add file unchange, than commit change if needed.

About instructions for Docker on Raspbian (i.e. Hypriot): I would keep them after explaining the context. Why not have instructions for someone who installs AGILE on an existing GW?